aggregator/src/aggregator.rs

```diff
@@ -3568,6 +3627,16 @@ where
                     Err(Error::Message(_)) => StatusCode::INTERNAL_SERVER_ERROR.into_response(),
                     Err(Error::HttpClient(_)) => StatusCode::INTERNAL_SERVER_ERROR.into_response(),
                     Err(Error::Http { .. }) => StatusCode::INTERNAL_SERVER_ERROR.into_response(),
+                    Err(Error::AggregateShareRequestRejected(task_id, _)) => {
+                        build_problem_details_response(
+                            DapProblemType::UnrecognizedMessage,
```

Comment:
Q. None of the existing DAP problem types really fit (I somewhat-arbitrarily chose `unrecognizedMessage`), but I think it's still useful to return a problem details response here. Thoughts on defining a custom error type here?

Reply:
We generally use `unrecognizedMessage` for problems parsing messages, whereas this case is a well-encoded message that contains a value we didn't like.

If the leader encounters this error, we don't expect it to do anything but give up on the aggregate share request, right? If so, then nothing more specific than HTTP 400 is needed. Maybe what we should do is change DAP to generalize its `reportRejected` error type into a broader `requestRejected` error type, and eventually use that. In the meamtime, I think `StatusCode::CLIENT_ERROR.into_response()` should suffice here (DAP-03 allows not providing an error document; https://datatracker.ietf.org/doc/html/draft-ietf-ppm-dap-03#section-3.2-2)